pkg/controller/migmigration/task.go

```diff
@@ -574,6 +629,53 @@ func (t *Task) Run() error {
 		if err = t.next(); err != nil {
 			return liberr.Wrap(err)
 		}
+	case CreateDirectVolumeMigration:
+		err := t.createDirectVolumeMigration()
+		if err != nil {
+			return liberr.Wrap(err)
+		}
+		if err = t.next(); err != nil {
+			return liberr.Wrap(err)
+		}
+	case DirectVolumeMigrationStarted:
+		dvm, err := t.getDirectVolumeMigration()
+		if err != nil {
+			return liberr.Wrap(err)
+		}
+		// Make sure it exists
+		if dvm == nil {
+			return errors.New("direct volume migration not found")
+		}
+		// FIXME: currently a placefiller
+		if err = t.next(); err != nil {
+			return liberr.Wrap(err)
+		}
+	case WaitForDirectVolumeMigrationToComplete:
+		dvm, err := t.getDirectVolumeMigration()
+		if err != nil {
+			return liberr.Wrap(err)
+		}
+		// if no dvm, continue to next task
```

Comment:
Having no DVM is OK? I assumed this phase would only run if direct migration flags are set and volumes exist.

Reply:
This will run if direct volume migration is selected. If there are no filesystem copy volumes to create, I'm assuming we won't have a DVM, so the "wait for complete" step will always move on.